components/grid/grid.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="author" content="Naveen Chandar">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../../css/reset.css">
+    <link rel="stylesheet" href="../../css/variables.css">
+    <link rel="stylesheet" href="../../css/common.css">
+    <link rel="stylesheet" href="./grid.css">
+    <title>Grid</title>
+</head>
+
+<body>
+    <section>
+        <div class="grid_container">
+            <div class="grid_group grid_two_column">
+                <div class="flex_row grid grid_one">Grid one</div>
+                <div class="flex_row grid grid_two">Grid two</div>
+            </div>
+        </div>
+        <br />
+        <div class="grid_container">
+            <div class="grid_group grid_three_column">
+                <div class="flex_row grid grid_one">Grid one</div>
+                <div class="flex_row grid grid_two">Grid two</div>
+                <div class="flex_row grid_three">Grid three</div>
```

Comment:
What's the use of class flex_row?

Reply:
common class for flex. Inorder to prevent repeated code, few flex classes has been created and utilising it wherever needed.